get_who_theme.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+
+import pandas as pd
+import requests
+
+
+def main():
+    # Run like `python get_who_theme.py "Overweight and Obesity"`
+    parser = argparse.ArgumentParser()
+    parser.add_argument("theme_name")
+    args = parser.parse_args()
+
+    requested_indicators = pd.read_csv("requestedIndicators.csv")
+
+    theme = get_theme(args.theme_name, requested_indicators)
+    transform_theme(theme)
+    write_excel(theme)
+    write_csv(theme)
+
+
+def get_theme(theme_name: str, requested_indicators: pd.DataFrame) -> pd.DataFrame:
+    theme = pd.DataFrame()
+
+    theme_indicators = requested_indicators[
+        (requested_indicators["Theme"] == theme_name)
+        & (requested_indicators["HostingReference"] == "WHO")
+    ]
+    theme_indicators = theme_indicators.drop_duplicates("IndicatorCodeDS")
+
+    for _, indicator in theme_indicators.iterrows():
+        indicator_data = get_indicator_data(indicator["IndicatorCodeDS"])
+        indicator_values = get_indicator_values(indicator_data, indicator)
+        theme = theme.append(indicator_values)
```

Comment:
This method of looping is more "Pythonic" than `for i in range(...)`, by iterating over objects instead of accessing objects with an index variable. The `_` is a placeholder for the row index value, since [`iterrows`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.iterrows.html) returns a tuple of `(index, Series)`.